webui/react/src/pages/TaskListPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import TaskList from 'components/TaskList';
+
+const TaskListPage: React.FC = () => {
+  return <TaskList workspaceId={1} />;
```

Comment:
moved `TaskList` to components folder since it is not for page anymore. Instead, I make a new component, `TaskListPage`, for the page folder.